manifests/check.pp

```diff
@@ -85,11 +85,12 @@
 
   $escaped_id = regsubst($id,'\/','_','G')
   File[$::consul::config_dir]
-  -> file { "${consul::config_dir}/check_${escaped_id}.json":
+  -> file { "${consul::config_dir}/check_${escaped_id}.json" :
```

Comment:
unless this is a new change in the style, can you remove the space before the colon?